### ISSUE TITLE
(high) lacking of easy check for "too much color contrast"

### ISSUE THREAD

**Issue by JediLin** [w3c/wai-easy-checks#166]
Location: 
page: Color Contrast - Easy Checks. Subsection: [Who depends on Color Contrast?](https://deploy-preview-131--wai-easychecks.netlify.app/test-evaluate/easy-checks/color-contrast/#who-depends-on-color-contrast)

Current wording:
- Some people need lower contrast as the highest contrast (black on white) might cause too much glare

Issue:
The following Checking session does not provide any information about how to check this need.
